data/scrape/lib/generate_summary.py

```diff
@@ -77,15 +79,70 @@ def main() -> None:
         # TSV and README have different first column.
         summaries.append([file_path] + row)
         readme_list.append([f"[TSV](tsv/{file_path})"] + row)
+
     # Sorts by path to TSV.
     summaries.sort(key=operator.itemgetter(0))
     readme_list.sort(key=operator.itemgetter(0))
-    # Writes the TSV.
+
+    # Writes the TSV and computes KPIs as it goes.
     with open(LANGUAGES_SUMMARY_PATH, "w", encoding="utf-8") as sink:
         tsv_writer = csv.writer(sink, delimiter="\t", lineterminator="\n")
         tsv_writer.writerows(summaries)
-    # Writes the README.
+
+    # Calculate KPIs
+    num_languages = len(unique_iso_codes)
+    num_scripts = len(unique_scripts)
+
+    # Calculate the number of "Broad" and "Narrow" files for unique iso codes and dialects
+    table = pd.read_csv(
+        LANGUAGES_SUMMARY_PATH,
+        sep="\t",
+        names=[
+            "link",
+            "iso_639_2_code",
+            "iso_639_language_name",
+            "wiktionary_language_name",
+            "script",
+            "dialect",
+            "filtered",
+            "narrow_broad",
+            "case_folding",
+            "no_entries",
+        ],
+    )
+    df = table[table["dialect"] != ""]
+    grouped_counts = (
+        df.groupby(["iso_639_2_code", "dialect", "narrow_broad"])
+        .size()
+        .unstack(fill_value=0)
+    )
+    narrow_pairs_files = grouped_counts["Narrow"].sum()
+    broad_pairs_files = grouped_counts["Broad"].sum()
+
+    # Calculate remaining KPIs
+    num_broad_files = table["narrow_broad"].value_counts()["Broad"]
+    num_narrow_files = table["narrow_broad"].value_counts()["Narrow"]
+    dialects = table.groupby(["iso_639_2_code", "dialect"]).ngroups
+    num_entries = table[table["filtered"] == False][
+        "no_entries"
+    ].sum()  # Count unfiltered pronunciations
```

Comment:
The comment at the end of this line has caused the line to wrap. Please put the comment on the previous line and reflow. 

(`black` ought to do this automatically but alas.)